.github/workflows/build_and_test.yml

```diff
@@ -514,7 +515,7 @@ jobs:
 
   create-test-report:
     needs: [ check-permissions, regress-tests, coverage-report, benchmarks, build-buildtools-image ]
-    if: ${{ !cancelled() && contains(fromJSON('["skipped", "success"]'), needs.check-permissions.result) }}
+    if: ${{ !cancelled() && contains(fromJSON('["skipped", "success"]'), needs.check-permissions.result) && github.event.action != 'ready_for_review' }}
```

Comment:
And probably here.

Reply:
This one reruns even if the earlier jobs were skipped hence to avoid rerunning we need to add this check